docs/items/globallootmodifiers.md

```diff
@@ -7,133 +7,142 @@ Registering a Global Loot Modifier
 -------------------------------
 
 You will need 4 things:
-1. Create a `global_loot_modifiers.json` file at `/data/forge/loot_modifiers/`.
-    This will tell Forge about your modifiers and works similar to [tags][].
+1. Create a `global_loot_modifiers.json`.
+    This will tell Forge about your modifiers and works similar to [tags].
 2. A serialized json representing your modifier.
     This will contain all of the data about your modification and allows data packs to tweak your effect.
 3. A class that extends `IGlobalLootModifier`.
     The operational code that makes your modifier work. Most modders can extend `LootModifier` as it supplies base functionality.
-4. Finally, the serializer for your operational class.
+4. Finally, a class that extends `GlobalLootModifierSerializer` for your operational class.
     This is [registered] as any other `IForgeRegistryEntry`.
 
 The `global_loot_modifiers.json`
 -------------------------------
 
-All you need to add here are the file names of your loot modifiers. The [ResourceLocation][resloc] specified points to `data/<namespace>/loot_modifiers/<path>.json`.
+The `global_loot_modifiers.json` represents all loot modifiers to be loaded into the game. This file must be placed within `data/forge/loot_modifiers/global_loot_modifiers.json`.
```

Comment:
Would it be worth reinforcing the idea that the file _must_ be under the `forge` namespace rather than under the mod's namespace?